eZ/Publish/API/Repository/PermissionService.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
+
+use eZ\Publish\API\Repository\Values\User\UserReference;
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * This service provides methods for permission related operations.
+ */
+interface PermissionService
+{
+    /**
+     * Get current user.
+     *
+     * Loads the full user object if not already loaded, if you only need to know user id use {@see getCurrentUserReference()}
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User
+     */
+    public function getCurrentUser();
+
+    /**
+     * Get current user reference.
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\UserReference
+     */
+    public function getCurrentUserReference();
+
+    /**
+     * Sets the current user to the given $user.
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\UserReference $user
+     */
+    public function setCurrentUser(UserReference $user);
+
+    /**
+     * @param string $module The module, aka controller identifier to check permissions on
+     * @param string $function The function, aka the controller action to check permissions on
+     * @param \eZ\Publish\API\Repository\Values\User\UserReference $user
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/b4c4758c7f1a4718d547eb18277d550d69e6b462, also added some description while at it.